src/pkg/fault/fault.go

```diff
@@ -40,9 +40,12 @@ type ErrorsData struct {
 // New constructs a new error with default values in place.
 func New(failFast bool) *Errors {
 	return &Errors{
-		mu:       &sync.Mutex{},
-		errs:     []error{},
-		failFast: failFast,
+		mu:   &sync.Mutex{},
+		errs: []error{},
+		// TODO: respect the failfast input.  But while we're
+		// in transition towards fault support, and not completely
+		// checking errs, it's better for us to catch everything.
+		failFast: true,
```

Comment:
we need to make sure this is well documented and folks know about it until we change it. Otherwise it becomes a bit confusing if people just look at the function signature

Reply:
Yup.  Per comments in sync, I will probably revert this but find a way to run in failFast for standard behavior.